apps/google-analytics-4/lambda/src/middlewares/allowCorsOptionsRequests.ts

```diff
@@ -0,0 +1,22 @@
+import { RequestHandler } from 'express';
```

Comment:
Add TODO to use express' cors library or is that something we won't do for this app?